srcpkgs/gdb/template

```diff
@@ -1,65 +1,59 @@
 # Template file for 'gdb'
 pkgname=gdb
 version=10.1
-revision=2
+revision=3
 build_style=gnu-configure
 pycompile_dirs="/usr/share/gdb/python"
-configure_args="--disable-werror --disable-nls --with-system-readline
```

Comment:
Please don't remove `--disable-werror`, thanks.

Reply:
Ok, added it back.